READMEs/developers.md

```diff
@@ -1,14 +1,16 @@
 # Developing ocean.py
 
 This README is how to further *develop* ocean.py. (Compare to the quickstarts which show how to *use* it.)
-
 Steps:
 1. **Install dependencies**
-1. **Start blockchain service** (only needed for ganache)
-1. **Deploy** the contracts to {local, rinkeby, mainnet}
-1. **Test**
+2. **Configure the services**
```

Comment:
Don't make it "2" and "3". Leave it as "1", since GitHub auto-numbers for you. Keeps things more maintainable.